### DISCUSSION TITLE
Hi, The screenshots don't upload when the user drag and drop the file in the "Screenshots" field

### DISCUSSION THREAD

**Discussion by radz1will** [directus/directus#6281] [Feature Requests]
The screenshots don't upload when the user drag and drop the file in the "Screenshots" field

Actual result:
The screenshots don't upload when the user drag and drop the file in the "Screenshots" field on the "Editing Item " page

Expected result:
The screenshots should be upload when the user drag and drop the file in the "Screenshots" field on the "Editing Item" page

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/discussions/4794
